frontend/src/components/PublishTags.tsx

```diff
@@ -63,7 +63,7 @@ const PublishTags = ({ blogId, onClick }: { blogId: string; onClick: () => Promi
 				onClick={async () => {
 					setLoading(true);
 					await onClick();
-					setVisibility(true);
+					setVisibility(title.length > 0 && content.length > 0);
```

Comment:
`title.trim().length > 0 & content.trim().length > 0` will take care of "   " scenarios

Reply:
Reconsider using visibility state. It is possible to perform these checks even without it.

Reply:
oh you mean an if statement or something like `title.trim().length > 0 & content.trim().length > 0 & setVisibility(true)`

Reply:
Yes. But I realise now that you are using it close the topics popup as well on line 124.
In that case, we can keep it as it is currently.

Just `trim()` is missing.